crates/gen-spidermonkey/spidermonkey-wasm/Makefile

```diff
@@ -0,0 +1,142 @@
+SM_REPO := https://github.com/fitzgen/gecko-dev
```

Comment:
Oh one thing I forgot to ask, would it make sense to build this on CI and ensure that `spidermonkey.wasm` in the repo is the same?

Reply:
In an ideal world, yes. Do we want to spend that CI time though?